contracts/StreamManager.vy

```diff
@@ -223,13 +223,12 @@ def cancel_stream(
     assert self.streams[creator][stream_id].start_time + MIN_STREAM_LIFE <= block.timestamp
 
     funded_amount: uint256 = self.streams[creator][stream_id].funded_amount
-    amount_locked: uint256 = funded_amount  - self._amount_unlocked(creator, stream_id)
+    amount_locked: uint256 = funded_amount - self._amount_unlocked(creator, stream_id)
+    self.streams[creator][stream_id].funded_amount = funded_amount - amount_locked
```

Comment:
Is there a case where `funded_amount` is 0 and `amount_locked` is also 0, so like canceling an empty stream, maybe by accident ( same operation twice in a block?).
Should be assert the stream is funded before canceling, I guess is what im wondering

Reply:
[ce90246](https://github.com/ApeWorX/ApePay/pull/100/commits/ce90246a521c814ea8807092bed7818de03268e8)